packages/client/components/example-component/stories.tsx

```diff
@@ -0,0 +1,43 @@
+import { storiesOf } from '@storybook/react'
```

Comment:
If we use `stories.tsx`, I think we should also use `test.tsx`. (Modify `testMatch` in `jest.config.ts`)